src/sg/gov/tech/geekout/1.0/schema.json

```diff
@@ -1,99 +1,65 @@
 {
-  "$id": "opencerts/v1.0",
+  "$id": "https://schemata.openattestation.com/sg/gov/tech/geekout/1.0",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
-    "issuedOn": {
-      "type": "string"
-    },
-    "expiredOn": {
-      "type": "string"
-    },
-    "name": {
-      "type": "string"
-    },
-    "issuer": {
+    "signatory": {
       "type": "object",
       "properties": {
         "name": {
           "type": "string"
         },
-        "did": {
-          "type": "string"
-        },
-        "url": {
+        "position": {
           "type": "string"
         },
-        "email": {
+        "organisation": {
           "type": "string"
         },
-        "phone": {
-          "type": "string"
-        },
-        "certificateStore": {
+        "signature": {
           "type": "string"
         }
       },
-      "required": ["name", "certificateStore"],
-      "additionalProperties": false
+      "required": [
+        "name",
+        "position",
+        "organisation",
+        "signature"
+      ]
     },
     "recipient": {
       "type": "object",
       "properties": {
         "name": {
           "type": "string"
-        },
-        "did": {
-          "type": "string"
-        },
-        "url": {
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "programme": {
+      "type": "object",
+      "properties": {
+        "name": {
           "type": "string"
         },
-        "email": {
+        "startDate": {
           "type": "string"
```

Comment:
format: date https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times